packages/commonwealth/client/scripts/views/pages/user_dashboard/TrendingCommunitiesPreview/TrendingCommunitiesPreview.tsx

```diff
@@ -12,13 +12,24 @@ export const TrendingCommunitiesPreview = () => {
   const navigate = useCommonNavigate();
   const user = useUserStore();
 
-  const { data } = useGetNewContent({ enabled: user.isLoggedIn });
+  const { data: newContent } = useGetNewContent({ enabled: user.isLoggedIn });
+  const { data: paginatedTrendingCommunities } = useFetchCommunitiesQuery({
+    cursor: 1,
+    limit: 3,
+    ...(user.isLoggedIn && {
+      relevance_by: 'membership',
+    }),
+    include_last_30_day_thread_count: true,
+    order_by: 'last_30_day_thread_count',
+    order_direction: 'DESC',
+  });
 
-  // TODO: https://github.com/hicommonwealth/commonwealth/issues/8760
-  const { data: activeCommunities } = useFetchActiveCommunitiesQuery();
```

Comment:
it is used also in `communities.tsx` file but I can see that in another PR you dont use it anymore. So in this or the other PR we should remove this hook

Reply:
Created a cleanup ticket: https://github.com/hicommonwealth/commonwealth/issues/8966